docker-php-ext-enable

```diff
@@ -40,7 +41,7 @@ fi
 for module in "${modules[@]}"; do
 	if grep -q zend_extension_entry "$module"; then
 		# https://wiki.php.net/internals/extensions#loading_zend_extensions
-		line="zend_extension=$module"
+		line="zend_extension=$EXT_DIR/$module"
```

Comment:
Instead of doing things with `EXT_DIR`, can we instead just change this line to the following:
```bash
		line="zend_extension=$(readlink -f "$module")"
```